src/docs/dynamic-sampling/fidelity-and-biases.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Fidelity and Biases
+sidebar_order: 2
+---
+
+Dynamic Sampling is a feature that allows Sentry to automatically adjust the amount of data retained based on the value of the data. This is technically achieved by applying a **sample rate** to every event, which is determined by a **set of rules** that are evaluated for each event.
+
+<Alert title="✨ Note" level="info">
+
+A sample rate is a number in the interval `[0.0, 1.0]` that will determine the likelihood of a transaction to be retained. For example, a sample rate of `0.5` means that the transaction will be retained on average 50% of the time.
+
+</Alert>
+
+## The Concept of Fidelity
+
+At the core of Dynamic Sampling there is the concept of **fidelity**, which translates to an overall target sample rate that should be applied across all transactions of an organization. Within this target sample rate, Dynamic Sampling can create a **bias toward more meaningful data**. This is achieved by constantly updating and communicating special rules to Relay, via a project configuration, which then applies targeted sampling to every event.
```

Comment:
The thing is that the definition of that number is billing specific, I will generally say that this can be determined on your own and at Sentry we use our own system.